tests/behaviour/typeql/language/steps.rs

```diff
@@ -400,4 +402,9 @@ generic_step_impl! {
             matched entries of given {actual_answers}."
         );
     }
+
+    #[step(expr = "set time-zone is: {word}")]
+    async fn time_zone_info(context: &mut Context, timezone: String) {
```

Comment:
Names of the BDD steps follow the wording of the step. This should be `set_time_zone`.

Also, why is there an "is" in the step? It should really read: "set time zone: {}".